src/Game/zGoo.h

```diff
@@ -30,6 +30,7 @@ struct zFXGooInstance
     float32 warbc[4];
     float32 state_time[4];
     xVec3 center;
+    int32 padding; // Padding.
```

Comment:
It'd be helpful to instead have the comment say something like: `// added padding to match function [insert function name]` so if someone messes with it in the future they know what it's there for